MediaSessionExt.cpp

```diff
@@ -12,14 +12,28 @@ using namespace WPEFramework;
 using SafeCriticalSection = Core::SafeSyncType<Core::CriticalSection>;
 extern Core::CriticalSection drmAppContextMutex_;
 
+#ifdef NETFLIX
```

Comment:
PR_3_3 instead?

Reply:
As this is the extended media session, thought that NETFLIX extender has a different macro definition. So I can go with your suggestion as this definition changes only in playready version 2.5 and 3.3